### PR TITLE
fix(linux): Prevent exception if neither USER, LOGNAME nor SUDO_USER set

### LINKED ISSUE #9542: bug(linux): Exception in `verify_ibus_daemon` if neither USER nor SUDO_USER set
On one system we got an exception in `verify_ibus_daemon`. It looks like neither USER, LOGNAME nor SUDO_USER were set to a non-empty string.

### PR DESCRIPTION
**Pull request by ermshiperete**
Closes #9542.

@keymanapp-test-bot skip

### CODE REVIEW

File: linux/keyman-config/keyman_config/ibus_util.py [hunk 1]
```diff
@@ -84,6 +84,10 @@ def restart_ibus_subp():
 def verify_ibus_daemon(start):
     realuser = os.environ.get('SUDO_USER')
     user = realuser or getpass.getuser()
+
+    if not user:
+        return
```
Comment (darcywong00) [inline linux/keyman-config/keyman_config/ibus_util.py:89]: Do we need to log?

Comment (ermshiperete) [reply linux/keyman-config/keyman_config/ibus_util.py:89]: Good idea. Done.

**Review by darcywong00 — APPROVED** [review]
lgtm

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 17.0.172-alpha](https://keyman.com/downloads/releases/alpha/17.0.172)

**Pull request merged**